### PR TITLE
Add nymphs to the game

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Cutie spreading weeds

## Why It's Good For The Game

Cute targets for marines, usefull for xeno.
WIP

## Changelog
:cl:
add: Add nymphs to the game, cute auto weeders that comes from nymphs nest
/:cl:

### CODE REVIEW

**Comment by FosterGrime-Heymynameslouie** [comment]
can you select what kind of weeds they spew out?

also the dawn of lesser beings serving xenos is now nigh

speded mergh instantly

**Comment by Autisem** [comment]
any idea what can create these nests? or that still being brain stormed?

**Comment by BraveMole** [comment]
> any idea what can create these nests? or that still being brain stormed?

Probably psy points

**Comment by lbnesquik** [comment]
Will they induce notable amounts of lag?

**Comment by BraveMole** [comment]
> Will they induce notable amounts of lag?

It is just a minion

File: code/modules/xenomorph/xeno_structures.dm [hunk 1]
```diff
@@ -1621,3 +1618,20 @@ TUNNEL
...
+	bound_width = 96
+	bound_height = 96
+	resistance_flags = UNACIDABLE | DROPSHIP_IMMUNE
+	xeno_structure_flags = IGNORE_WEED_REMOVAL | CRITICAL_STRUCTURE
+
+/obj/structure/xeno/nymph_nest/Initialize()
+	. = ..()
+	SSspawning.registerspawner(src, INFINITY, list(/mob/living/carbon/xenomorph/nymph/ai), 0, 0, null)
+	SSspawning.spawnerdata[src].required_increment = 2 * max(45 SECONDS, 3 MINUTES - SSmonitor.maximum_connected_players_count * SPAWN_RATE_PER_PLAYER)/SSspawning.wait
+	SSspawning.spawnerdata[src].max_allowed_mobs = max(1, MAX_SPAWNABLE_MOB_PER_PLAYER * SSmonitor.maximum_connected_players_count * 2)
```
Comment (TiviPlus) [inline code/modules/xenomorph/xeno_structures.dm:1637]: why arent these args?

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by lKiseki** [comment]
I have come to make an announcement. They've taken us for fools, brothers and sisters.
image
Do you find this cute? Do you truly think this is CUTE?

image
DOES THIS LOOK LIKE THE FACE OF MERCY TO YOU?

**Review by Kuro020 — APPROVED** [review]

**Comment by Oscari-pog** [comment]
Those are whole fucking ants man I thought it was gonna be like cute lil' mini benos

**Comment by BraveMole** [comment]
> Those are whole fucking ants man I thought it was gonna be like cute lil' mini benos

They are cute shutup

**Pull request closed without merging**